crates/sui-config/src/node.rs

```diff
@@ -1282,7 +1282,7 @@ fn default_max_transaction_manager_queue_length() -> usize {
 }
 
 fn default_max_transaction_manager_per_object_queue_length() -> usize {
-    20
+    2000
```

Comment:
this seems very high, but i guess the idea is we would never hit this before we hit the age check anyway?

Reply:
exactly. originally i was thinking we should delete this, but I figured it would be worth keeping as a "protect us if something went very wrong" backstop-backstop